hathor/transaction/base_transaction.py

```diff
@@ -481,22 +496,39 @@ def validate_basic(self, skip_block_weight_verification: bool = False) -> bool:
         meta.validation = ValidationState.BASIC
         return True
 
-    def validate_full(self, skip_block_weight_verification: bool = False) -> bool:
+    def validate_full(self, skip_block_weight_verification: bool = False, sync_checkpoints: bool = False) -> bool:
         """ Run full validations (these need access to all dependencies) and update the validation state.
 
         If no exception is raised, the ValidationState will end up as `FULL` and return `True`.
```

Comment:
It might end up as `CHECKPOINT_FULL`, right?

Reply:
Yes, I'll include that.

Reply:
Fixed in #292 .